contrib/cirrus/setup_environment.sh

```diff
@@ -448,3 +463,5 @@ echo -e "\n# End of global variable definitions" \
 
 msg "Global CI Environment vars.:"
 grep -Ev '^#' /etc/ci_environment | sort | indent
+
+showrun echo "my work here is done"
```

Comment:
```suggestion
msg "finished"
```

For consistency and ctrl-f ease, would you consider making this similar to the final message as in `runner.sh`?

Reply:
Done, except I kept `showrun`